examples/PaylaterInvoice/index.php

```diff
@@ -0,0 +1,146 @@
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Unzer UI Examples</title>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"
+            integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="
+            crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://static.unzer.com/v1/unzer.css" />
+    <script type="text/javascript" src="https://static.unzer.com/v1/unzer.js"></script>
+</head>
+
+<body style="margin: 70px 70px 0;">
+
+<p><a href="https://docs.unzer.com/reference/test-data" target="_blank">Click here to open our test data in new tab.</a></p>
+
+<form id="payment-form">
+    <div id="customer" class="field">
+        <!-- The customer form UI element will be inserted here -->
+    </div>
+    <div id="example-invoice-secured"></div>
+    <div class="field" id="error-holder" style="color: #9f3a38"></div>
+    <button class="unzerUI primary button fluid" id="submit-button" type="submit">Pay</button>
+</form>
+
+<script>
+    // Create an Unzer instance with your public key
+    let unzerInstance = new unzer('<?php echo UNZER_PAPI_PUBLIC_KEY; ?>');
+
+    // Create an Invoice Secured instance
+    let paylaterInvoice = unzerInstance.PaylaterInvoice();
+    paylaterInvoice.create({
+        containerId: 'example-invoice-secured',
+        errorHolderId: 'error-holder'
+    });
+
+    // Create a customer instance and render the customer form
+    let Customer = unzerInstance.Customer();
+    Customer.create({
+        containerId: 'customer',
+        differentBillingAddress: true,
+        horizontalDivider: true,
+        errorHolderId: 'error-holder',
+        paymentTypeName: 'paylater-invoice'
+    });
+
+    // Handle payment form submission.
+    let form = document.getElementById('payment-form');
+    let payButton = document.getElementById("submit-button");
+    payButton.disabled = true;
+
+    let validCustomer = false;
+    let validResource = false;
+    paylaterInvoice.addEventListener('change', function eventHandlerResource(e) {
+        if (e.optinSuccess) {
+            validResource = true;
+            if (validCustomer) {
+                $('button[type="submit"]').removeAttr('disabled');
+            }
+        } else {
+            validResource = false;
+            $('button[type="submit"]').attr('disabled', 'disabled');
+        }
+    })
+
+    Customer.addEventListener('validate', function eventHandlerCustomer(e) {
+        if (e.success) {
+            validCustomer = true;
+            if (validResource) {
+                $('button[type="submit"]').removeAttr('disabled');
+            }
+        } else {
+            $('button[type="submit"]').attr('disabled', 'disabled');
+            validCustomer = false;
+        }
+    })
+
+    form.addEventListener('submit', function(event) {
+        event.preventDefault();
+        let PaylaterInvoicePromise = paylaterInvoice.createResource();
```

Comment:
camelCase to be used for variable names (PaylaterInvoicePromise -> paylaterInvoicePromise)

Reply:
fixed in PR #80